src/UI/Components/Diagram/init.ts

```diff
@@ -230,7 +227,10 @@ export default function diagramInit(
       );
 
       if (doesTargetHaveRule) {
-        targetCell.addRelation(sourceCell.id as string, sourceName);
+        targetCell.addRelation(
+          sourceCell.id as string,
+          doesTargetHaveRule.attrName as string,
```

Comment:
`doesTargetHaveRule` is a boolean value for me when I read the variable name. Why is it an object?